src/mono/wasm/build/WasmApp.targets

```diff
@@ -316,8 +318,6 @@
      <_WasmRuntimePackNativeLibs Include="libmono-profiler-aot.a"/>
      <_WasmRuntimePackNativeLibs Include="libicuuc.a" />
      <_WasmRuntimePackNativeLibs Include="libicui18n.a" />
-     <_WasmObjects Include="@(_WasmRuntimePackNativeLibs->'$(MicrosoftNetCoreAppRuntimePackRidDir)\native\%(FileName)%(Extension)')" />
-
```

Comment:
Why was this removed?

Reply:
The same files were added a few lines later.

Reply:
🤦‍♂️